dev/small-requirements.txt

```diff
@@ -1,26 +1,17 @@
 ## Small test reqs
-# Required
-azure-storage-blob==12.3.0
-# Required to log artifacts and models to GCS artifact locations
-google-cloud-storage==1.14.0
-# Required to log artifacts and models to AWS S3 artifact locations
-botocore==1.12.84  # pinned for moto
-boto3==1.9.84      # pinned for moto
-# Required to log artifacts and models to HDFS artifact locations
-pyarrow==0.17.0
 # Required to log artifacts to SFTP artifact locations
-pysftp==0.2.9
+pysftp
 # Required to run the MLflow server against SQL-backed storage
-sqlalchemy==1.3.0
+sqlalchemy
```

Comment:
This is a core dependency that's already specified in `setup.py` and should be removed from this file